main.py

```
@@ -41,34 +41,126 @@ def run(self):
+                self.crawler.normalized_urls[normalized_url] = current_url
+                self.crawler.visited_urls.add(current_url)
+                self.crawler.stats['total_crawled'] += 1
+                
+                # コンテンツを変換・保存
+                file_path = self.converter.process_page(current_url, html_content)
+                if file_path:
+                    success_count += 1
+                    print(f"  → 保存: {file_path}")
+                else:
+                    failed_count += 1
+                    print(f"  → 変換失敗")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)